Dockerfile

```diff
@@ -0,0 +1,38 @@
+# golang:1.16 image created 2021-06-24T00:31:06.02014601Z 
+FROM docker.io/library/golang@sha256:be99fa59acd78bb22a41bbc1e15ebfab2262498ee0c2e28c3d09bc44d51d1774 AS builder
+
+# Build the preflight binary
+WORKDIR /go/src/preflight
+COPY . .
```

Comment:
Looks good. Slight suggestion. Could we invert line 5 and 6 and make the copy more prescriptive. Feels a little puzzling in the current state.

```
COPY . /go/src/preflight
WORKDIR /go/src/preflight
```

Reply:
Reordered the commands to be more explicit.